src/main/java/domain/Cars.java

```diff
@@ -0,0 +1,60 @@
+package domain;
+
+import util.Exceptions;
+
+import java.util.*;
+
+public class Cars {
+
+    private final List<Car> cars;
+
+    public Cars(List<String> names) {
+        validateDuplicatedNames(names);
+        cars = convertNamesToCars(names);
+    }
+
+    private void validateDuplicatedNames(List<String> names) {
+        if (names.size() != Set.copyOf(names).size()) {
+            throw new IllegalArgumentException(Exceptions.DUPLICATED_NAME_EXCEPTION.getMessage());
+        }
+    }
+
+    private List<Car> convertNamesToCars(List<String> names) {
+        List<Car> cars = new ArrayList<>();
+
+        for (String name : names) {
+            cars.add(new Car(name));
+        }
+
+        return cars;
+    }
+
+    public List<String> findWinners() {
+        List<Car> winners = cars.stream()
+                .sorted(Comparator.comparing(Car::getForward).reversed())
+                .takeWhile(car -> car.getForward() == getMaxForward())
+                .toList();
```

Comment:
```suggestion
        int maxForward = getMaxForward();
        List<Car> winners = cars.stream()
                .filter(car -> car.getForward() == maxForward)
                .toList();
```
filter를 사용하면 훨씬 쉽져!? stream도 많이 사용하다보면 다양한 메서드들을 알게 되실 거예요!
저는 일단 `.`을 찍어두고 나오는 메서드들을 주욱 훑어보면서 적절한 메서드를 찾는 방식을 많이 사용합니다. (안 되면 GPT..)

<img width="584" alt="image" src="https://github.com/woowacourse/java-racingcar/assets/77425729/ef739ed6-dfcf-43af-bc49-716e06a8a819">

Reply:
stream을 사용할 때 늘 쓰던 메서드만 써서 다른 메서드를 고려하지 못한 것 같습니다!
앞으로 미션을 진행할 때는 적합한 메서드가 있을지 살펴보는 습관을 들일 수 있도록 노력하도록 하겠습니다 🙇